src/settings.ts

```diff
@@ -48,6 +50,7 @@ export const DEFAULT_SETTINGS: DataviewSettings = {
         inlineQueryPrefix: "=",
         inlineJsQueryPrefix: "$=",
         enableDataviewJs: true,
+        taskLinkText: "=>",
```

Comment:
I would copy-paste some fancy link symbol like 🔗.

Reply:
I tried using emoji characters in the regex for task annotations, but they didn't seem to work. Do you have examples where emoji are in use elsewhere in the codebase today?

Reply:
Emoji characters are a pain in the ass. Look at `expression/parse.ts`, specifically on how tag and identifier parsing is implemented for a way to get it to work.

Reply:
Yeah, it was annoying to deal with emoji in the regex for dates, so I'm kind of surprised to see something like this. I wonder if there's a build setting that simplifies their use? https://github.com/schemar/obsidian-tasks/blob/main/src/Task.ts#L294

Reply:
Possibly; it mostly comes down to bad Unicode support for most Regex engines (and it varies by OS, to make things worse). You may be able to do it by looking up the exact emoji codepoints and using the unicode `\u+...` syntax in a regex.

Reply:
Yeah, that's what I wound up doing in the task regex. I was just surprised their emoji character works directly. 🤷 I'll figure it out!